### PR TITLE
Feature/frontend add logo types

### PR DESCRIPTION
**Pull request by styczynski**
Merge all features into starter-auto branch.

### CODE REVIEW

File: frontend/src/components/universal/CardDescription/index.tsx [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+interface WrapperProps {
+  fontSize?: string;
+  theme?: any;
+};
+
+const Wrapper = styled.div`
+  font-family: ${(props: WrapperProps) => props.theme.primaryFont};
+  color: ${(props: WrapperProps) => props.theme.colors.accent};
+  font-family: ${(props: WrapperProps) => props.theme.primaryFont};
```
Comment (mbalc) [inline frontend/src/components/universal/CardDescription/index.tsx]: why have this line duplicated?

File: frontend/src/components/universal/CardDescription/index.tsx [hunk 2]
```diff
@@ -0,0 +1,29 @@
+import * as React from 'react';
+import styled  from 'styled-components';
+
+interface WrapperProps {
+  fontSize?: string;
+  theme?: any;
```
Comment (mbalc) [inline frontend/src/components/universal/CardDescription/index.tsx]: not sure, but can this be `required`?

File: frontend/src/components/universal/CardTitle/index.tsx [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+interface WrapperProps {
+  fontSize?: string;
+  theme?: any;
+};
+
+const Wrapper = styled.div<WrapperProps>`
+  font-family: ${(props: WrapperProps) => props.theme.primaryFont};
+  color: ${(props: WrapperProps) => props.theme.colors.accent};
+  font-family: ${(props: WrapperProps) => props.theme.primaryFont};
```
Comment (mbalc) [inline frontend/src/components/universal/CardTitle/index.tsx]: same here

File: frontend/src/components/universal/CardDescription/index.tsx (hunk 1 above)
Comment (styczynski) [reply frontend/src/components/universal/CardDescription/index.tsx]: I am not entirely sure, but I'll be fixed.

File: frontend/src/components/universal/CardTitle/index.tsx (hunk 3 above)
Comment (styczynski) [reply frontend/src/components/universal/CardTitle/index.tsx]: One occurrence to be removed.

File: frontend/src/components/universal/CardDescription/index.tsx (hunk 2 above)
Comment (styczynski) [reply frontend/src/components/universal/CardDescription/index.tsx]: Seems like an option to me.

Comment (styczynski) [reply frontend/src/components/universal/CardDescription/index.tsx]: I'll change that.

**Comment by styczynski** [comment]
Merging branch after changes requested by the review were implemented.

**Pull request merged**